nextcord/client.py

```diff
@@ -1959,6 +2039,37 @@ def add_cog(self, cog: ClientCog) -> None:
             self.add_application_command(app_cmd, use_rollout=True)
         self._client_cogs.add(cog)
 
+    def add_application_check(self, func: ApplicationCheck) -> None:
+        """Adds a global application check to the bot.
+
+        This is the non-decorator interface to :meth:`.check`
+        and :meth:`.check_once`.
+
+        Parameters
+        -----------
+        func
+            The function that was used as a global application check.
+        """
+
+        self._connection._application_checks.append(func)
+
+    def remove_application_check(self, func: ApplicationCheck) -> None:
+        """Removes a global check from the bot.
+
+        This function is idempotent and will not raise an exception
+        if the function is not in the global checks.
+
+        Parameters
+        -----------
+        func
+            The function to remove from the global application checks.
+        """
+
+        try:
+            self._connection._application_checks.remove(func)
+        except ValueError:
+            pass
```

Comment:
Again the naming here leaves a lot to be desired, they're not application checks, they're application command checks, and should be named as such. This comment applies to anywhere else in the source these names are used.